spec/fixtures/renamed-file.git/hooks/applypatch-msg.sample

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
Are all of these hook files in the fixtures entirely necessary for the test to fun? If at all possible to prone them down, we should 👍

Reply:
I was just doing that. It bothered me so much to see all those extra unnecessary files! :)